admins.md

```diff
@@ -0,0 +1,11 @@
+ # QuTiP Admin Team
+
+The functioning of the admin team of the QuTiP project can be found in the [governance document](governance.md#2-admin-team). The current members of the admin team are:
+
+- Shahnawaz Ahmed
+- Eric Giguere
```

Comment:
is Eric's name missing some diacritics?

Reply:
Eric Gigu**è**re, but I don't really mind.

Reply:
we may as well do it right, I suppose!